apps/publikator/components/editor/modules/author/ui.js

```diff
@@ -50,6 +52,8 @@ export default ({ TYPE, newBlock, editorOptions }) => {
               portrait: author.value.portrait,
               credentials: author.value.credentials,
             },
+            isLarge: author.value.isLarge,
```

Comment:
just to be sure I understand correctly: `isLarge` and `greeting` are independent of each other, right? `isLarge` can be false and the component still displays the greeting

Reply:
they are independent of each other, however, the greeting is only displayed in the "isLarge" case